compiler/rustc_middle/src/mir/traversal.rs

```diff
@@ -232,6 +233,90 @@ pub fn postorder<'a, 'tcx>(
     reverse_postorder(body).rev()
 }
 
+struct UsedLocals(BitSet<Local>);
```

Comment:
Can this struct be merged with the one in simplify_locals?

Reply:
I found a change to non_ssa_locals that makes it possible to obviate this live locals tracking :)

Reply:
I see that comment was very unclear. Yes, I've already pushed the change in question to this PR.